rpcserver.go

```diff
@@ -5563,17 +5563,18 @@ func handleTxFeeInfo(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (
 // handleValidateAddress implements the validateaddress command.
 func handleValidateAddress(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
 	c := cmd.(*dcrjson.ValidateAddressCmd)
-
 	result := dcrjson.ValidateAddressChainResult{}
 	addr, err := dcrutil.DecodeAddress(c.Address)
 	if err != nil {
 		// Return the default value (false) for IsValid.
 		return result, nil
```

Comment:
is this correct?  should it return the error?

Reply:
It definitely should not return an error.